app/views/events/show.html.erb

```diff
@@ -1,6 +1,18 @@
 <h1><%= @event.name %></h1>
-<subtitle><%= date_range(@event) %></subtitle>
-<p role="doc-subtitle">
+<p role="doc-subtitle"><%= date_range(@event) %></p>
+<% if current_user&.profile&.attending? @event %>
```

Comment:
On line 9, we are doing `@current_user.profile`, so it implies that `current_user` should never be `nil`, which means we don't need to use the coalescing operator